scripts/build/builders/openiotsdk.py

```diff
@@ -72,6 +77,12 @@ def _build(self):
         self._Execute(['bash', '-c', cmd],
```

Comment:
This was already here, but wondering: why do we delete outputs to force re-linking. The comment says "force linking" ... but why?  this means that this does a `ld` on every build regardless if I have changes or not ... that seems wrong.

Reply:
Remnant of the first implementation, I will remove it.